src/components/WebPlayBack.tsx

```diff
@@ -119,13 +121,13 @@ function WebPlayback(props: IWebPlaybackProps) {
 
     return (
         <div className="WebPlayback">
-            <Search />
+            {deviceId && <Search deviceId = {deviceId}/>}
```

Comment:
We have a few components now that are dependent on making a connection to the Spotify APIs for us to render them. At some point, let's refactor this to see if we want to move this state that many components are dependent on into Redux + display some sort of loading indicator as we fetch the information we need to render our entire application

Reply:
Thanks for the reminder.